qabelbox/src/main/java/de/qabel/qabelbox/contacts/view/adapters/ContactDetailsAdapter.kt

```diff
@@ -1,46 +1,48 @@
 package de.qabel.qabelbox.contacts.view.adapters
 
+import android.view.View
 import android.widget.Button
 import de.qabel.core.config.Identity
 import de.qabel.qabelbox.R
 import de.qabel.qabelbox.contacts.dto.ContactDto
-import de.qabel.qabelbox.contacts.extensions.*
-import de.qabel.qabelbox.contacts.view.views.ContactDetailsFragment
+import de.qabel.qabelbox.contacts.extensions.contactColors
+import de.qabel.qabelbox.contacts.extensions.initials
+import de.qabel.qabelbox.contacts.extensions.readableKey
+import de.qabel.qabelbox.contacts.extensions.readableUrl
 import de.qabel.qabelbox.contacts.view.widgets.ContactIconDrawable
-import kotlinx.android.synthetic.main.fragment_contact_details.*
-import org.jetbrains.anko.ctx
+import kotlinx.android.synthetic.main.fragment_contact_details.view.*
 import org.jetbrains.anko.layoutInflater
 
 
 class ContactDetailsAdapter(private val onSendMessageClick: (identity: Identity) -> Unit) {
 
-    var view: ContactDetailsFragment? = null
+    var view: View? = null
 
     private var currentContact: ContactDto? = null
 
     fun loadContact(contact: ContactDto) {
         currentContact = contact;
         view?.apply {
-            contact_icon_border.background = ContactIconDrawable(contact.contactColors(ctx))
+            contact_icon_border.background = ContactIconDrawable(contact.contactColors(view!!.context))
             tv_initial.text = contact.initials();
             editTextContactName.text = contact.contact.alias;
             editTextContactDropURL.text = contact.readableUrl();
             editTextContactPublicKey.text = contact.readableKey();
 
             contact_details_actions.removeAllViews();
-            if (contact.identities.size > 0) {
+            if (contact.identities.size > 1 || (contact.identities.size == 1 && !contact.active)) {
                 contact.identities.forEach {
-                    addMessageButton(ctx.getString(R.string.send_message_as, it.alias));
+                    addMessageButton(view!!.context.getString(R.string.send_message_as, it.alias));
                 }
-            } else {
-                addMessageButton(ctx.getString(R.string.btn_chat_send, contact.identities.first().alias))
+            } else if (contact.identities.size == 1) {
+                addMessageButton(view!!.context.getString(R.string.send_message, contact.identities.first().alias))
             }
         }
     }
 
     private fun addMessageButton(text: String?) {
         view?.apply {
-            val button = ctx.layoutInflater.inflate(R.layout.contact_details_message_button, null) as Button;
+            val button = view!!.context.layoutInflater.inflate(R.layout.contact_details_message_button, null) as Button;
```

Comment:
apply applies the lambda as an extension method. The non-nullable view is now "this" in that scope.

Take a look at Any.let(), I think that is what you want to use instead.